### DISCUSSION TITLE
about Supabase Settings and Logs

### DISCUSSION THREAD

**Discussion by kim-das** [supabase/supabase#27987] [Questions]
![화면 캡처 2024-07-15 171412](https://github.com/user-attachments/assets/0b68476f-35c6-4996-8de6-182a122f2a43)
![화면 캡처 2024-07-15 171432](https://github.com/user-attachments/assets/afa154d1-84dc-409f-8801-87c0bcb3e439)
![화면 캡처 2024-07-15 171509](https://github.com/user-attachments/assets/0fe43699-b5a5-41d1-a1f8-f91009a47b1b)

I'm going to reach out to the staff. Please translate my question into English and show me the Korean translation for confirmation.

"I have checked DB/AUTH/STORAGE, and it seems like there is more usage than what I have used. I would like to request verification.

STORAGE: I have only created empty folders and haven't used them. However, logs continue to be generated.
DATABASE: Even though I only declared data names and specified data types, logs are being generated. Especially in the logs, I see rest/vq, /auth/v1/health frequently, and also storage/v1/bucket, admin/v1/network-bans/retrieve, admin/v1/ssl-config/db/status. What are these logs?
I suspect these logs might not be from my usage but from someone else's. I would appreciate your help.

Also, please help me understand the meaning of these logs.

Lastly, please let me know how I can verify if the settings for projects like DB and STORAGE are set to private."

**Comment by GaryAustin1** [comment]
Most of the calls you show are the dashboard when you are using it.  They check the health of auth and the REST API for the database.  Also any screens you go to, for instance to show storage, will make API calls.
